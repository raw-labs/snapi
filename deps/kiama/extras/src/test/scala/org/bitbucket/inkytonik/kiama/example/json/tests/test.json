{
   "one"   : 1.7,
   "two"   : "two",
   "three" : true,
   "four"  : false,
   "five"  : null,
   "six"   : { "a" : 5, "b" : "hello" },
   "seven" : [ "c", 7e5, [ false, true ], { "d" : "D", "e" : "E" } ]
}
