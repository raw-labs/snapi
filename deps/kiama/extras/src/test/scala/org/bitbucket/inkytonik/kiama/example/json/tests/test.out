JObject(
 Vector(
  JName("one") -> JNumber(1.7),
  JName("two") -> JString("two"),
  JName("three") -> JTrue(),
  JName("four") -> JFalse(),
  JName("five") -> JNull(),
  JName("six") -> JObject(
   Vector(JName("a") -> JNumber(5.0), JName("b") -> JString("hello"))),
  JName("seven") -> JArray(
   Vector(
    JString("c"),
    JNumber(700000.0),
    JArray(Vector(JFalse(), JTrue())),
    JObject(Vector(JName("d") -> JString("D"), JName("e") -> JString("E")))))))
{
 "one" : 1.7,
 "two" : "two",
 "three" : true,
 "four" : false,
 "five" : null,
 "six" : { "a" : 5, "b" : "hello" },
 "seven" : [ "c", 700000, [ false, true ], { "d" : "D", "e" : "E" } ]
}
